src/query/test/test.go

```diff
@@ -0,0 +1,579 @@
+// Copyright (c) 2020 Uber Technologies, Inc.
```

Comment:
Turns out these should be `// Modifications Copyright (c) 2020 Uber Technologies, Inc.` , otherwise it looks good